diff --git a/system/stack/l2cap/l2c_fcr.cc b/system/stack/l2cap/l2c_fcr.cc
index cc0bc0569a..0d0e8e8caa 100644
--- a/system/stack/l2cap/l2c_fcr.cc
+++ b/system/stack/l2cap/l2c_fcr.cc
@@ -1593,13 +1593,6 @@ BT_HDR* l2c_lcc_get_next_xmit_sdu_seg(tL2C_CCB* p_ccb, bool* last_piece_of_sdu)
 uint8_t l2c_fcr_chk_chan_modes(tL2C_CCB* p_ccb) {
   log::assert_that(p_ccb != NULL, "assert failed: p_ccb != NULL");
 
-  /* Remove nonbasic options that the peer does not support */
-  if (!(p_ccb->p_lcb->peer_ext_fea & L2CAP_EXTFEA_ENH_RETRANS) &&
-      p_ccb->p_rcb->ertm_info.preferred_mode == L2CAP_FCR_ERTM_MODE) {
-    log::warn("L2CAP - Peer does not support our desired channel types");
-    p_ccb->p_rcb->ertm_info.preferred_mode = 0;
-    return false;
-  }
   return true;
 }
 
diff --git a/system/stack/l2cap/l2c_link.cc b/system/stack/l2cap/l2c_link.cc
index 07c576496b..83c892ead5 100644
--- a/system/stack/l2cap/l2c_link.cc
+++ b/system/stack/l2cap/l2c_link.cc
@@ -108,9 +108,6 @@ void l2c_link_hci_conn_comp(tHCI_STATUS status, uint16_t handle, const RawAddres
     /* Connected OK. Change state to connected */
     p_lcb->link_state = LST_CONNECTED;
 
-    /* Get the peer information if the l2cap flow-control/rtrans is supported */
-    l2cu_send_peer_info_req(p_lcb, L2CAP_EXTENDED_FEATURES_INFO_TYPE);
-
     if (p_lcb->IsBonding()) {
       log::debug("Link is dedicated bonding handle:0x{:04x}", p_lcb->Handle());
       if (l2cu_start_post_bond_timer(handle)) {
