diff --git a/60-steam-input.rules b/60-steam-input.rules
index 66afaee..1672d9f 100644
--- a/60-steam-input.rules
+++ b/60-steam-input.rules
@@ -228,3 +228,14 @@ SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="057e", ATTRS{idP
 
 # Nintendo Switch 2 GameCube Controller over USB
 SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="057e", ATTRS{idProduct}=="2073", MODE="0660", TAG+="uaccess"
+
+# From https://github.com/nirenjan/libx52/blob/master/udev/60-saitek-x52-x52pro.rules.in
+## X52Pro flight control system
+SUBSYSTEM=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0762", MODE="0777", TAG+="uaccess"
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0762", MODE="0777", TAG+="uaccess"
+
+## X52 flight control system
+SUBSYSTEM=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0255", MODE="0777", TAG+="uaccess"
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0255", MODE="0777", TAG+="uaccess"
+SUBSYSTEM=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="075c", MODE="0777", TAG+="uaccess"
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="075c", MODE="0777", TAG+="uaccess"
